policy-packs/aws-ts-finops/PulumiPolicy.yaml

```diff
@@ -0,0 +1,2 @@
+description: A collection of finops policies for AWS
```

Comment:
```suggestion
description: A collection of FinOps policies for AWS
```

Pedantic, however, an homage to the Pulumi team member that coined the term!

Reply:
I updated it. It should have been FinOps from the beginning.